code/modules/mob/living/living_defines.dm

```diff
@@ -55,6 +70,6 @@
 	var/bubble_icon = "default" //what icon the mob uses for speechbubbles
 
 	var/last_bumped = 0
-	var/unique_name = 0 //if a mob's name should be appended with an id when created e.g. Mob (666)
+	var/unique_name = 0 //if a mob's name should be appended with an id when created e.g. Mob (69)

```

Comment:
Don't change that, mobs have a THREE digit ID, your comment implies that IDs are 2 digit.